src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -36,14 +36,9 @@
 #include "ngsiNotify/Notifier.h"
 
 
-
-/* ****************************************************************************
-*
-* Select 'method' to send notifications - only one can be uncommented
-*/
-//#define SEND_BLOCKING
-#define SEND_IN_NEW_THREAD
-
+// FIME P11 #1669: move this to WS library. @fortizc please take care of this
+#define WS_SCHEME         "ws://"
+#define WS_SCHEME_LENGTH  5
```

Comment:
This is only a suggest, but may be is better use a static const std::string and to get its length you can use size() method, moreover when you assign a char * to std::string internally make something like that: std::string(my_char_pointer) in this way is better too a const std::string